crates/plonk-verifier-codegen/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "plonk-verifier-codegen"
+version.workspace = true
+edition = "2018"
+license = "MIT OR Apache-2.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+handlebars = "*"
+serde = "*"
+serde_derive = "*"
+serde_json = "*"
+hex = "*"   
```

Comment:
Please don't use wildcard dependencies -- packages with wildcard deps cannot be published on `crates.io`. As a rule of thumb, using `major.minor` is a good approach.